src/transformers/models/auto/modeling_auto.py

```diff
@@ -838,6 +841,15 @@
     ]
 )
 
+
+MODEL_FOR_AUDIO_VISUAL_CLASSIFICATION_MAPPING_NAMES = OrderedDict(
```

Comment:
I'm not sure there's a need for a new mapping here, since audio visual classification seems very specific.

cc'ing @sgugger here

Reply:
Yes, let's refrain from adding a new mapping just for one model, and in the model addition PR. This can be done later on, when we have several models with the same API.